src/platform-release.js

```diff
@@ -128,14 +127,32 @@ exports.prepareReleaseBranchCommand = function*() {
                '\n' +
                'Usage: $0 prepare-release-branch --version=3.6.0 -r platform')
     );
+   
     var repos = flagutil.computeReposFromFlag(argv.r);
-    var version = flagutil.validateVersionString(argv.version);
-    var branchName = getVersionBranchName(version);
+    var branchName = null;
+ 
 
     // First - perform precondition checks.
     yield repoupdate.updateRepos(repos, [], true);
 
     yield repoutil.forEachRepo(repos, function*(repo) {
+       
+        var version = null;
+
+        if (argv.version === undefined) {
+            // Grabbing version from platformPackageJson
```

Comment:
minor: can you fix the indenting? the else should be in the same vertical line as the if. comment should be inline with the vars, etc.

once you fix it, do a `git add platform-release.js` and then `git commit --amend`. This will allow you to add the change to the existing commit instead of creating a new commit. You will then have to push it with `git push audrey master --force`